java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/domain/Category.java

```diff
@@ -15,5 +18,20 @@
 
   private String description;
 
-  private List<Item> items;
+  @Builder.Default
+  private List<Item> items = new ArrayList<>();
+
+  public Category addUpdateItem(Item item) {
```

Comment:
It's not clear to me what this method's doing. Do we follow a naming convention?